### PR TITLE
fix broken links : http://kubernetes.io/gettingstarted/

### PR DESCRIPTION
**Pull request by huchengze**

### CODE REVIEW

File: content/en/blog/_posts/2015-10-00-Some-Things-You-Didnt-Know-About-Kubectl_28.md [hunk 1]
```diff
@@ -146,6 +146,6 @@ Ahh, that’s better.
 
 ##### Conclusion
 
-So there you have it, nine new and exciting things you can do with your Kubernetes cluster and the kubectl command line. If you’re just getting started with Kubernetes, check out [Google Container Engine](https://cloud.google.com/container-engine/) or other ways to [get started with Kubernetes](http://kubernetes.io/gettingstarted/).  
+So there you have it, nine new and exciting things you can do with your Kubernetes cluster and the kubectl command line. If you’re just getting started with Kubernetes, check out [Google Container Engine](https://cloud.google.com/container-engine/) or other ways to [get started with Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/).
```
Comment (tengqm) [inline content/en/blog/_posts/2015-10-00-Some-Things-You-Didnt-Know-About-Kubectl_28.md]:
```suggestion
So there you have it, nine new and exciting things you can do with your Kubernetes cluster and the kubectl command line. If you’re just getting started with Kubernetes, check out [Google Container Engine](https://cloud.google.com/container-engine/) or other ways to [get started with Kubernetes](/docs/tutorials/kubernetes-basics/).  
```

File: content/en/blog/_posts/2015-11-00-Kubernetes-1-1-Performance-Upgrades-Improved-Tooling-And-A-Growing-Community.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ As we mentioned above, we would love your help:
 - Connect with the community on [Slack](http://slack.kubernetes.io/)
 - Follow us on Twitter [@Kubernetesio](https://twitter.com/kubernetesio) for latest updates&nbsp;
 - Post questions (or answer questions) on Stackoverflow&nbsp;
-- Get started running, deploying, and using Kubernetes [guides](http://kubernetes.io/gettingstarted/)&nbsp;
+- Get started running, deploying, and using Kubernetes [guides](https://kubernetes.io/docs/tutorials/kubernetes-basics/);
```
Comment (tengqm) [inline content/en/blog/_posts/2015-11-00-Kubernetes-1-1-Performance-Upgrades-Improved-Tooling-And-A-Growing-Community.md]:
```suggestion
- Get started running, deploying, and using Kubernetes [guides](/docs/tutorials/kubernetes-basics/);
```

File: content/en/blog/_posts/2015-12-00-Managing-Kubernetes-Pods-Services-And-Replication-Controllers-With-Puppet.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ People familiar with [Puppet](https://github.com/puppetlabs/puppet)&nbsp;might h
 
 ### The Puppet Kubernetes Module
 
-The Puppet Kubernetes module currently assumes you already have a Kubernetes cluster [up and running](http://kubernetes.io/gettingstarted/).&nbsp;Its focus is on managing the resources in Kubernetes, like Pods, Replication Controllers and Services, not (yet) on managing the underlying kubelet or etcd services. Here’s a quick snippet of code describing a Pod in Puppet’s DSL.  
+The Puppet Kubernetes module currently assumes you already have a Kubernetes cluster [up and running](https://kubernetes.io/docs/tutorials/kubernetes-basics/);Its focus is on managing the resources in Kubernetes, like Pods, Replication Controllers and Services, not (yet) on managing the underlying kubelet or etcd services. Here’s a quick snippet of code describing a Pod in Puppet’s DSL.
```
Comment (tengqm) [inline content/en/blog/_posts/2015-12-00-Managing-Kubernetes-Pods-Services-And-Replication-Controllers-With-Puppet.md]:
```suggestion
The Puppet Kubernetes module currently assumes you already have a Kubernetes cluster [up and running](/docs/tutorials/kubernetes-basics/);Its focus is on managing the resources in Kubernetes, like Pods, Replication Controllers and Services, not (yet) on managing the underlying kubelet or etcd services. Here’s a quick snippet of code describing a Pod in Puppet’s DSL.  
```

**Comment by huchengze** [comment]
@tengqm Thank you.

**Pull request merged**